javalin/src/main/java/io/javalin/jetty/JettyServer.kt

```diff
@@ -55,6 +56,7 @@ class JettyServer(val config: JavalinConfig) {
             this.sessionHandler = config.inner.sessionHandler
             config.inner.servletContextHandlerConsumer?.accept(this)
             addServlet(ServletHolder(wsAndHttpServlet), "/*")
+            JettyWebSocketServletContainerInitializer.configure(this, null) // Initializes WebSocketComponents
```

Comment:
What's `this` and `null` ? What does this do?

Reply:
it's just a custom initializer to do some stuff during initialization, replaced this with empty lambda